sdk/attestation/attestation/src/generated/generatedClientContext.ts

```diff
@@ -7,25 +7,25 @@
  */
 
 import * as coreHttp from "@azure/core-http";
-import { AttestationClientOptionalParams } from "./models";
+import { GeneratedClientOptionalParams } from "./models";
 
 const packageName = "@azure/attestation";
-const packageVersion = "1.0.0-beta.2";
+const packageVersion = "1.0.0";
```

Comment:
Version 1.0.0-beta.3?

Reply:
That is generated code, from the swagger file.

How does JS/TS ensure that all three separate version numbers are kept in sync?